packages/@sanity/portable-text-editor/src/editor/Editable.tsx

```diff
@@ -444,6 +447,30 @@ export const PortableTextEditable = forwardRef(function PortableTextEditable(
     [onFocus, portableTextEditor, change$, slateEditor],
   )
 
+  const handleClick = useCallback(
+    (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+      if (onClick) {
+        onClick(event)
+      }
+      // Inserts a new block if it's clicking on the editor, focused on the last block and it's a void element
+      if (slateEditor.selection && event.target === event.currentTarget) {
```

Comment:
PK will likely be the best person, but I wonder if this could be something that was moved to a normalise function :)

Reply:
Curious to that answer, but I think it's not possible. as this is handling the onClick event.
It tries to solve the action in which the user clicks below the void block and nothing is added, so he can't continue typing.
The first part of the video is current behaviour, second part is with the changes. 

https://github.com/sanity-io/sanity/assets/46196328/4e604259-1eec-4f64-a1c3-1be969c21e3f

Reply:
It's a timely question @RitaDias!

However in this case I think we don't want that. We want it to be tied to the UI-event of the user trying to click of move below with the intention of creating a new block.

We don't in all cases want it to always insert a empty block there (that would mean it's impossible to end a editor with an object block).